.travis.yml

```diff
@@ -0,0 +1,21 @@
+## -*- coding: utf-8 -*-
+# 
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+
+sudo: false
+dist: trusty
+language: python
+python:
+  - 3.6
```

Comment:
Please add `2.7` for users who haven't made the jump to Python3 and `3.5` since that's the latest in Debian stable.